src/api/app/assets/javascripts/webui/application.js

```diff
@@ -36,6 +36,7 @@
 //= require webui/application/attribute
 //= require webui/application/main
 //= require webui/application/repository_tab
+//= require webui/application/moment
```

Comment:
We don't just drop unversioned javascript libraries into the codebase. Please use a rubygem like https://rubygems.org/gems/momentjs-rails